module/calcStylePoints.test.js

```diff
@@ -18,6 +18,14 @@ describe("calcStylePoints", () => {
     assert.equal(actual, expected);
   });
 
+  it("should return style notes for non-numeric notes", () => {
+    const actual = calcStylePoints(["18", 19.5, 19, 17, 18.5]);
+
+    const expected = NaN;
```

Comment:
you need to change your function, NaN isn't a value that you want to show users ;)